docs/error-reporting/web-console/debug.md

```diff
@@ -16,42 +16,45 @@ Below is a screenshot of the type of data you will see in the Debugger tool.
 
-### Callstack Section
-The callstack pane contains the callstack of the currently selected thread. Mousing over a frame will reveal additional interesting information about it. If a frame has an orange highlight, it is deemed more likely to be relevant to the crash.
+<img src={useBaseUrl('img/error-reporting/console-views/debug-view-breadcrumbs.png')} alt="Shows the Breadcrumbs pane in the Debug view." />
 
-### Global Variables Section
-This pane is present for the Backtrace snapshot format used on UNIX and UNIX-like systems. It contains the values of any global variables that have been serialized.
+The breadcrumbs that are added depend on the SDK that you're using. For more information on which events are automatically captured or how to configure breadcrumbs, see [Platform Integrations](/error-reporting/getting-started/).
 
-### Registers Section
-This pane is present for dump formats such as minidump. It contains the register values for the currently selected frame.
-
-### Thread Section
-This pane contains a list of all threads in the dump. Threads that have crashed have a red icon next to them. Threads that have an arrow symbol next to them can be expanded to more threads with an identical callstack. In other words, threads are grouped into a tree if they have identical callstacks in order to ease navigating lots of threads.
+### Global Variables
+This pane is present for the Backtrace snapshot format used on UNIX and UNIX-like systems. It contains the values of any global variables that have been serialized.
 
-### Variable Context Section
+### Variable Context
 This pane is present for the Backtrace snapshot format used on UNIX and UNIX-like systems. It contains auxiliary information about a variable including type information, memory allocator state (size of allocation a variable points to, whether it is active or freed and much more).
 
-### Variable Section
+### Variables
 This pane is present for the Backtrace snapshot format used on UNIX and UNIX-like systems. Variables with a red box are currently freed. Variables with a green box are currently allocated. Variables with a red circle have warnings attached to them, click on the variable to expand it and see any potential warnings.
 
-### Warnings Section
+### Warnings
 This pane is present for the Backtrace snapshot format used on UNIX and UNIX-like systems. Use the prev and next button to browse warnings. Press the Select button to jump to a warning. Alternatively, check the Autoselect variable button to automatically jump to a variable that has a warning attached to it. Warnings include things like invalid memory allocator state, automated detection of variables accessed when the crash occurred and more.
```

Comment:
Update format for using interface elements.
```suggestion
This pane is present for the Backtrace snapshot format used on UNIX and UNIX-like systems. Use the prev and next button to browse warnings. Click **Select** to jump to a warning. Alternatively, click **Autoselect variable** to automatically jump to a variable that has a warning attached to it. Warnings include things like invalid memory allocator state, automated detection of variables accessed when the crash occurred and more.
```

Reply:
Change "Warnings include things like invalid memory allocator state, automated detection of variables accessed when the crash occurred and more" to "Warnings may include invalid memory allocator state, automated detection of variables accessed when the crash occurred, and more"